src/assets/styles/theme.scss

```diff
@@ -59,4 +59,5 @@ $BCgovFontColorInverted: #ffffff;
 
 // App Colors
 $app-blue: #1669bb; // same as the Vuetify theme primary
-$app-red: #d3272c; 
+$app-red: #d3272c;
+$app-green: #1a9031,
```

Comment:
Add "same as" to red and green, please.

Reply:
What are they the same as? I'm not entirely sure the red is the same as the theme but will confirm.

Reply:
Lol. In the theme you said they were the same as these :)

Reply:
The primary is for sure, although i'm not certain about the others.

Reply:
Ok.. i see what you mean 🤦  lol, Updated!